thealgorithms/project_euler/problem_36/sol1.py

```diff
@@ -8,23 +8,27 @@
 
 (Please note that the palindromic number, in either base, may not include leading zeros.)
 '''
-try:
-	xrange		#Python 2
-except NameError:
-	xrange = range	#Python 3
+def main():
+    try:
+	    xrange		#Python 2
+    except NameError:
+	    xrange = range	#Python 3
 
-def is_palindrome(n):
-	n = str(n)
+    def is_palindrome(n):
```

Comment:
Would be nice if this function was declared outside of main

Reply:
True I didnt see this one